### PR TITLE
Fixes #1147 add missing info

### LINKED ISSUE #1147: Observer details to be shown in PKSim
Some details are missing here:
* Is imported observer Amount-Observer or Container-Observer
* Is defined for all molecules or not
* Molecules-IncludeList (if defined not for all molecules)
* Molecules-ExcludeList (if defined for all molecules)
* Dimension

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.UI/Views/Observers/ObserverInfoView.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ public void BindTo(ObserverDTO observerDTO)
       {
          Clear();
          observerDTO.Details.Each(x => richEditControl.Document.AppendHtmlText($"{x}<br>"));
+         richEditControl.Document.CaretPosition = richEditControl.Document.Range.Start;
```
Comment (msevestre) [inline src/PKSim.UI/Views/Observers/ObserverInfoView.cs]: this makes sure we are always at the top!

File: src/PKSim.Assets/PKSimConstants.cs [hunk 2]
```diff
@@ -2172,6 +2173,10 @@ public static class UI
          public static readonly string WatermarkText = "Text";
          public static readonly string WatermarkProperties = "Watermark Properties";
          public static readonly string CreateTableSolubilityAlternative = "Create as pH-Solubility table";
+         public static readonly string ContainerCriteria = "Container Criteria";
+         public static readonly string ForAll = "For All";
+         public static readonly string MoleculeExcludeList = "Molecules Excluded";
```
Comment (msevestre) [inline src/PKSim.Assets/PKSimConstants.cs]: Could be also For All Molecules Except

File: src/PKSim.Assets/PKSimConstants.cs [hunk 3]
```diff
@@ -2172,6 +2173,10 @@ public static class UI
          public static readonly string WatermarkText = "Text";
          public static readonly string WatermarkProperties = "Watermark Properties";
          public static readonly string CreateTableSolubilityAlternative = "Create as pH-Solubility table";
+         public static readonly string ContainerCriteria = "Container Criteria";
+         public static readonly string ForAll = "For All";
```
Comment (msevestre) [inline src/PKSim.Assets/PKSimConstants.cs]: Could be For All Molecules

File: src/PKSim.Assets/PKSimConstants.cs (hunk 2 above)
Comment (Yuri05) [reply src/PKSim.Assets/PKSimConstants.cs]: same: better with _molecules_

File: src/PKSim.Assets/PKSimConstants.cs (hunk 3 above)
Comment (msevestre) [reply src/PKSim.Assets/PKSimConstants.cs]: ok. BTW: Should the text be For All Molecules or For all molecules?

File: src/PKSim.Assets/PKSimConstants.cs (hunk 2 above)
Comment (Yuri05) [reply src/PKSim.Assets/PKSimConstants.cs]: What you proposed: `For All Molecules Except ... `

Comment (msevestre) [reply src/PKSim.Assets/PKSimConstants.cs]: done

Comment (msevestre) [reply src/PKSim.Assets/PKSimConstants.cs]: done

**Pull request merged**